src/lib/consensus/proof_of_stake.ml

```diff
@@ -1500,6 +1330,63 @@ module Data = struct
     end
   end
 
+  module Consensus_time = struct
+    open Graphql_async
+    open Schema
+    include Global_slot
+
+    let graphql_type () =
+      let open Graphql_lib.Base_types in
+      obj "ConsensusTime" ~fields:(fun _ ->
+          [ field "epoch"
+              ~typ:(non_null @@ uint32 ())
+              ~args:Arg.[]
+              ~resolve:(fun _ global_slot -> epoch global_slot)
+          ; field "slot"
```

Comment:
do you think it's worth putting `globalSlot` as it's own field as well? Seems like we're talking about it more and more. We can always add it later too.